server/models/UsersModel.mjs

```diff
@@ -35,6 +35,16 @@ class UsersModel {
       createdAt: admin.firestore.FieldValue.serverTimestamp(),
     };
   }
+
+  static async update(uid, updates) {
+    const db = admin.firestore();
+    try {
+      await db.collection('users').doc(uid).update(updates);
+    } catch (error) {
+      console.error('Error updating user:', error);
+      throw error;
+    }
+  }
```

Comment:
I think this update() should be in userController.mjs file not in UsersModel.mjs file.

Reply:
addressed!